src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -154,17 +157,25 @@ private static void analyzeRepos(String outputPath, List<RepoConfiguration> conf
             } catch (RuntimeException e) {
                 logger.log(Level.SEVERE, String.format(MESSAGE_BRANCH_DOES_NOT_EXIST,
                         config.getBranch(), config.getLocation()), e);
-                generateEmptyRepoReport(repoReportDirectory.toString());
+                generateEmptyRepoReport(repoReportDirectory.toString(), Author.NAME_FAILED_TO_CLONE_OR_CHECKOUT);
                 continue;
             }
-            analyzeRepo(config, repoReportDirectory.toString());
+
+            try {
+                analyzeRepo(config, repoReportDirectory.toString());
```

Comment:
To simplify some code, the analyzeRepo can just be combined in the try catch above.